bin/figma2flutter.dart

```diff
@@ -112,16 +134,87 @@ List<TokenTheme> _processTokens(List<TokenTheme> themes) {
   return processor.themes;
 }
 
+/// Loads all the design token info into a single structure resulting in
+/// something that looks like it would if it was all in one file
+///
+/// Creates a single JSON strcture from the driven by the $metadata.json
+/// Add the $metadata.json and $themes.json
+Map<String, dynamic> _arrangeJsonFilesBySection(String inputFileLocation) {
```

Comment:
Would love to have a test that covers this method. Maybe you can refactor it to a class thats available during testing?

Reply:
Moved the $theme and $metadata manipulation code into lib and wrote tests.  None of your current tests read from files so I wasn't sure how you wanted to arrange that.